ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -376,8 +383,12 @@ const Annotations = React.memo(({ variant, mainGeneId, showMainGene }) => {
   } = variant
   const mainTranscript = getVariantMainTranscript(variant)
 
-  const lofDetails = (mainTranscript.lof === 'LC' || mainTranscript.lofFlags === 'NAGNAG_SITE') ? [
-    ...(mainTranscript.lofFilter ? [...new Set(mainTranscript.lofFilter.split(/&|,/g))] : []).map((lofFilterKey) => {
+  const isLofNagnag = mainTranscript.isLofNagnag || mainTranscript.lofFlags === 'NAGNAG_SITE'
+  const lofFilters = mainTranscript.lofFilters || (
+    mainTranscript.lof === 'LC' && mainTranscript.lofFilter && mainTranscript.lofFilter.split(/&|,/g)
+  )
```

Comment:
It looks like a duplicate of the loading pipeline. Can we simply use `mainTranscript.isLofNagnag` and `mainTranscript.lofFiters`?

Reply:
As I state in the PR description: we should support both v2 and v3 in the UI

So no